dynamics-nav/SETRECORD-Function--Page-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETRECORD Function (Page)"
+description: SETRECORD Function (Page) Sets the current record for the page.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main